src/routes.ts

```diff
@@ -134,4 +136,16 @@ export abstract class Routes {
         },
       }),
   }
+
+  public readonly connectedAccounts = {
+    list: () =>
+      this.makeRequest<ConnectedAccountsListResponse>({
+        url: "/connected_accounts/list",
+      }),
+
+    get: (connected_account_id: string) =>
+      this.makeRequest<ConnectedAccountsGetResponse>({
+        url: `/connected_accounts/get?connected_account_id=${connected_account_id}`,
```

Comment:
Minor nitpick but could you use `params: { ... }` here like the other GET requests instead of putting the parameter directly in the URL?